openmrs/src/test/java/org/motechproject/openmrs/tasks/impl/OpenMRSActionProxyServiceTest.java

```diff
@@ -190,7 +190,7 @@ public void shouldCreateEncounterWithGivenParametersWithObsWithManyValues() {
 
         DateTime encounterDatetime = new DateTime("2000-08-16T07:22:05Z");
         Map<String, String> observations = new HashMap<>();
-        observations.put("testConceptName","testObservationValueName0, testObservationValueName1");
+        observations.put("testConceptName","testObservationValueName0, ,testObservationValueName1");
```

Comment:
it easy to lose this empty value, can you add a brief java comment that empty value should be not included